### PR TITLE
[16.0][IMP] web_refresher: Charge t-inherit-mode extension views just after the main

### PR DESCRIPTION
**Pull request by CarlosRoca13**
By doing this, the extension view is charged just after the main, so if some other views are using this other, to make a primary one, like project is doing for project.task control_panel, the refresher will continue be shown.

Before the patch:
image

After the patch:
image

cc @Tecnativa TT48551

ping @chienandalu @pedrobaeza

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Great! :+1:

File: web_refresher/__manifest__.py [hunk 1]
```diff
@@ -11,7 +11,17 @@
         "web.assets_backend": [
             "web_refresher/static/src/**/*.scss",
             "web_refresher/static/src/**/*.esm.js",
-            "web_refresher/static/src/**/*.xml",
+            "web_refresher/static/src/xml/refresher.xml",
+            (
```
Comment (pedrobaeza) [inline web_refresher/__manifest__.py:18]: Always explain the whys

```suggestion
            # Load the modification of the master template just after it,
            # for having the modification in all the primary extensions.
            # Example: the project primary view.
            (
```

**Comment by CarlosRoca13** [comment]
Added

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**